ClimsoftVer4/ClimsoftVer4/ucrFormDaily2.vb

```diff
@@ -161,10 +163,12 @@ Public Class ucrFormDaily2
                             ctr.Focus()
                         End If
                     End If
+                    If ctrVFP.Tag = iMonthLength Then
+                        ucrInputTotal.GetFocus()
+                    End If
```

Comment:
I have now used this code block to move the focus to the totals field when the last active date is reached.

Reply:
Now finding a way to get it to the save from the big control